src/platform/utilities/feature-toggles/featureFlagNames.js

```diff
@@ -30,6 +30,7 @@ export default Object.freeze({
   vaOnlineSchedulingProviderSelection: 'vaOnlineSchedulingProviderSelection',
   vaOnlineSchedulingCheetah: 'vaOnlineSchedulingCheetah',
   vaGlobalDowntimeNotification: 'vaGlobalDowntimeNotification',
+  vaOnlineSchedulingHomepageRefresh: 'vaOnlineSchedulingHomepageRefresh',
```

Comment:
Is there a vets-api PR to add this feature flag yet?

Reply:
Done.